hal/us_ticker_api.h

```diff
@@ -31,6 +31,15 @@ extern "C" {
  * @{
  */
 
+typedef void (*TICKER_IRQ_HANDLER_TYPE)(const ticker_data_t *const);
```

Comment:
please for typedef use small letters, as any other implementation does in this codebase

Reply:
fixed.